application/play/scala/example/.g8/form/app/controllers/$model__Camel$Controller.scala

```diff
@@ -0,0 +1,46 @@
+package controllers
+
+import javax.inject._
+import play.api.mvc._
+
+import play.api.data._
+import play.api.data.Forms._
+
+case class $model;format="Camel"$Data(name: String, age: Int)
+
+// NOTE: Add the following to conf/routes to enable compilation of this class:
+/*
+GET     /$model;format="camel"$        controllers.$model;format="Camel"$Controller.$model;format="camel"$Get
+POST    /$model;format="camel"$        controllers.$model;format="Camel"$Controller.$model;format="camel"$Post
+*/
```

Comment:
Is this comment auto-generated, or is this an action that the user must take?

If it's the latter I'd say we mention it in the README.

If it's not necessary I'd say we remove it, since it might come across as something that needs to be done, based on the phrasing.

Reply:
These files are generated by the template, but they are not used for our case. 
Therefore, I removed them and all the files from `.g8` directory